x-pack/plugins/security/server/config.ts

```diff
@@ -150,6 +150,9 @@ export const ConfigSchema = schema.object({
     lifespan: schema.nullable(schema.duration()),
   }),
   secureCookies: schema.boolean({ defaultValue: false }),
+  sameSiteCookies: schema.maybe(
```

Comment:
I follow the same name convention as `secureCookies` does. Let me know if we want to rename it or group cookie-related settings under a namespace (can be done separately).

Reply:
This is fine for now. I created https://github.com/elastic/kibana/issues/68237 for us to introduce a grouping